WindowsAppRuntime.sln

```diff
@@ -301,7 +301,11 @@ EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "AppNotifications", "dev\AppNotifications\AppNotifications.vcxitems", "{B4824897-88E0-4927-8FB9-E60106F01ED9}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Windows.AppNotifications.Projection", "dev\Projections\CS\Microsoft.Windows.AppNotifications.Projection\Microsoft.Windows.AppNotifications.Projection.csproj", "{2385A443-A1C2-4562-8D28-D0AD239EE5E7}"
+EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Common", "Common", "{82A73181-EA4A-431A-B82B-BE6734604CC9}"
+	ProjectSection(SolutionItems) = preProject
+		SDKPackageManager.ps1 = SDKPackageManager.ps1
```

Comment:
Pre-Project?

For Common?

So you want this to run for everyone using WindowsAppRuntime.sln, regardless the feature they're working on?

When errors occur how are they detected and handled?